app/src/main/java/com/github/kulebin/myfoursquareapp/app/HttpRequestInterceptor.java

```diff
@@ -0,0 +1,32 @@
+package com.github.kulebin.myfoursquareapp.app;
+
+import android.net.Uri;
+
+import com.github.kulebin.myfoursquareapp.BuildConfig;
+import com.github.kulebin.myfoursquareapp.http.HttpRequest;
+import com.github.kulebin.myfoursquareapp.http.IInterceptor;
+
+class HttpRequestInterceptor implements IInterceptor.IRequestIntercept {
+
+    private static final String PARAM_CLIENT_ID = "client_id";
+    private static final String PARAM_CLIENT_SECRET = "client_secret";
+
+    @Override
+    public HttpRequest interceptRequest(final HttpRequest request) {
+        return new HttpRequest.Builder()
```

Comment:
method signature has been changed to "void interceptRequest(final HttpRequest request)",
in such case there is no need to return anything